build/Dockerfile.j2

```diff
@@ -25,6 +31,8 @@
 
 FROM base as final
 
+USER easy_infra
```

Comment:
This ensures that before the `Dockerfrag`s take effect, the user will always be `easy_infra`.